specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,102 @@
+| Analytics                 | Analytics Platform                | Unified solution that combines technologies to meet enterprise needs across the entire analytics lifecycle.                                                                                            |
+|                           | Business Intelligence             | Semantic models, dashboards, reports, and data visualizations to track performance and identify trends.                                                                                                |
+|                           | Data Processing                   | Integration and transformation tasks to prepare data for analysis.                                                                                                                                     |
+|                           | Search                            | Discovery of information by indexing and retrieving data from various sources.                                                                                                                         |
+|                           | Streaming Analytics               | Real-time data stream processes to detect patterns, trends, and anomalies as they occur.                                                                                                               |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                             |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                    |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                                                                                                       |
+|                           | End User Computing                | Virtualized desktop infrastructure and device / endpoint management.                                                                                                                                   |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Enablement of compute capabilities without provisioning or managing servers.                                                                                                                           |
```

Comment:
Functions is a marketing name. Serverless compute captures the types of services we're trying to capture perfectly. 

Can you elaborate what you mean by 'nearly every PaaS offering could fit into it'? Are the core function of those services to provide 'compute'? I would think not since they're PaaS. We debated this already but IMO, its not relevant if databases for example user serverless compute. They're under the Database category and aren't getting captured in Compute for a service-level. We're not mixing up SKU level categorization with this (SQL server cost would only be in Database / Relational - not in compute/storage/network/dabaase etc. even though that service uses all of these).

Reply:
Aligned on path forward, which is to leave as-is.  Resolving.